### PR TITLE
OSOE-588: Upgrade to 1.5.0

### PR DESCRIPTION
**Pull request by jtkech**
[OSOE-588](https://lombiq.atlassian.net/browse/OSOE-588)

Start upgrading to OC 1.5.0

Related https://github.com/Smithsonian/CFCH-folkwaysCore/pull/92
https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions/issues/424

[OSOE-588]: https://lombiq.atlassian.net/browse/OSOE-588?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by Piedone** [comment]
We'd need this merged before OC 1.6 is released and we start upgrading to that. When do you think you can finish here?

**Comment by jtkech** [comment]
For me it was ready but I will look at #38 this week end.

**Comment by Piedone** [comment]
Thanks!

**Comment by Piedone** [comment]
Let me know here when you're ready.

**Comment by jtkech** [comment]
@Piedone

Okay, I did some last updates and retried it under https://github.com/Smithsonian/CFCH-folkwaysCore/pull/92 to migrate Folkways to OC 1.5.0, seems to be working. So I think it is ready to be reviewed.

File: Lombiq.ContentEditors/Tests/Lombiq.ContentEditors.Tests.UI/Lombiq.ContentEditors.Tests.UI.csproj
Comment (Piedone) [inline Lombiq.ContentEditors/Tests/Lombiq.ContentEditors.Tests.UI/Lombiq.ContentEditors.Tests.UI.csproj:1]: Move this project to the root of the repo instead of the subfolder under the module folder.

File: Lombiq.ContentEditors/Assets/Scripts/async-editor/async-editor.js [hunk 1]
```diff
@@ -226,12 +227,12 @@ window.asyncEditor.editor = {
...
 
             return editorGroups[index + 1];
-        }
+        },
     },
 };
 
 window.initAsyncEditor = (asyncEditorId, parameters) => {
-    if (!parameters) return window.asyncEditor.editors[asyncEditorId];
+    if (!parameters) return;
```
Comment (Piedone) [inline Lombiq.ContentEditors/Assets/Scripts/async-editor/async-editor.js:235]: Why is this change necessary?

**Review by Piedone — CHANGES_REQUESTED** [review]

File: Lombiq.ContentEditors/Tests/Lombiq.ContentEditors.Tests.UI/Lombiq.ContentEditors.Tests.UI.csproj
Comment (jtkech) [reply Lombiq.ContentEditors/Tests/Lombiq.ContentEditors.Tests.UI/Lombiq.ContentEditors.Tests.UI.csproj:1]: Okay

File: Lombiq.ContentEditors/Assets/Scripts/async-editor/async-editor.js (hunk 1 above)
Comment (jtkech) [reply Lombiq.ContentEditors/Assets/Scripts/async-editor/async-editor.js:235]: To fix an eslint error because here we were returning something, but if there are parameters, below we init `window.asyncEditor.editors[asyncEditorId]` but nothing is returned.

**Comment by jtkech** [comment]
@Piedone 

Okay updated.

Why did you remove pnpm-lock from .gitignore?

As it is generated on build

File: Lombiq.ContentEditors/Assets/Scripts/async-editor/async-editor.js (hunk 1 above)
Comment (Piedone) [reply Lombiq.ContentEditors/Assets/Scripts/async-editor/async-editor.js:235]: OK then.

**Comment by Piedone** [comment]
Lock files should be stored in the repos. They're regenerated but their purpose is to lock floating NPM package versions to specific ones, and if we don't store them in the repo then different developers may get different versions, which can cause different behavior.

**Review by Piedone — APPROVED** [review]
